CHANGELOG.md

```diff
@@ -37,6 +37,7 @@
 * Display `effective call number prefix`, `call number`, `call number suffix`, `enumeration`, `chronology`, `volume` in loans contexts. Refs UIU-1347, UIU-1391.
 * Prevent change due date for declared lost items. Refs UIU-1207.
 * Tweak text for declared lost modal. Refs UIU-1444.
+* Update eslint to >= 6.2.1 or eslint-util >= 1.4.1. Refs UIU-1446.
```

Comment:
There are a lot of provided changes before the release date, and there is a poor test code coverage level. Even stylish refactoring can cause some issues. So how can we be sure that there are not any broken changes?